15-eco.Rmd

```diff
@@ -7,12 +7,19 @@ In it you will also make use of R's\index{R} interfaces to dedicated GIS\index{G
 
 The chapter uses the following packages:
 
-```{r 14-eco-1, message=FALSE, eval=FALSE}
-library(sf)
-library(raster)
-# library(RQGIS)
-library(mlr)
+```{r 15-eco-1, message=FALSE, eval=FALSE}
+library(data.table)
 library(dplyr)
+library(mlr3)
```

Comment:
Same comment: I suggest not loading all of these, some can be used with `pkg::fun()`.

Reply:
This probably won't work. For example, you need to attach **mlr3spatiotempcv** to make available the spatial partitioning stuff used in `mlr3::rsmp()`. Equally, you need to attach **mlr3learners** and **mlr3extralearners** to make available additional learners. And the `autoplot()` function will only work with benchmark results if you have previoulsy attached **mlr3viz** to make the `autoplot()`. And there are more examples.